docs/tutorials/markdown-guide.md

```diff
@@ -189,7 +189,7 @@ Another Text
 * * *
 ### **Videos**
 
-``` markdown title="Sintaxis"
+``` markdown title="Syntax"
 <iframe width="100%" height="350" src="https://www.youtube.com/embed/tss1d0sow0o" frameborder="0" allow="true" allowfullscreen></iframe> 
 ```
 **Ejemplo:**
```

Comment:
**Example:**

Reply:
Can you check if it changed? I apologize for the delays I have had issues with GitHub.

Reply:
yes looks good!